collections-arrays/RemoveDuplicatesFromAnArray/BenchmarkRunner/RemoveDuplicateElementsBenchmark.cs

```diff
@@ -0,0 +1,65 @@
+﻿using BenchmarkDotNet.Attributes;
+using RemoveDuplicatesFromAnArray;
+
+namespace BenchmarkRunner
```

Comment:
The array should be filled just once. You can declare it as a private field and fill in the values once and reuse it in all the methods. Also, you don't need to assign the result to the response variable, no point in doing that.